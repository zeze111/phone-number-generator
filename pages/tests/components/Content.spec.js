import React from "react";
import { shallow } from "enzyme";
import Content from "../../components/content";
import NumbersList from "../../components/numbersList";

describe("Content", () => {
  it("should render the component", () => {
    const wrapper = shallow(<Content />);
    expect(wrapper).toMatchSnapshot();
  });

  describe("setLinkNo", () => {
    it("should set the state of linkNo", () => {
      const wrapper = shallow(<Content />);
      const link = 2;
      wrapper.setState({ linkNo: link });
      expect(wrapper.instance().state.linkNo).toBe(2);
    });
  });
});
