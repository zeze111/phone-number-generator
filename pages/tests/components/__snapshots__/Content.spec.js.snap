// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render the component 1`] = `
<div
  className=".body"
>
  <div
    className="sidebar-content"
  >
    <Sidebar
      linkNo={1}
      setLinkNo={[Function]}
    />
  </div>
  <div
    className="content"
  >
    <Generate />
  </div>
</div>
`;
